packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewYoutube.css

```diff
@@ -0,0 +1,3 @@
+@import 'c/quanticResultQuickview';
+@import 'c/quanticResultActionStyles';
```

Comment:
why exactly do we need the result actions styles here?

Reply:
good catch will remove